install.sh

```diff
@@ -212,80 +214,6 @@ libffi_install ()
     popd
 }
 
-coincurve_patch_ignore_sys_libsecp ()
-{
-    cat <<'EOF' > setup_support.py.patch
-74c74,77
-<         ffi.dlopen("secp256k1")
----
->         if "COINCURVE_IGNORE_SYSTEM_LIB" in os.environ:
->             return False
->         else:
->             ffi.dlopen("secp256k1")
-EOF
-    cat <<'EOF' > setup.py.patch
-216,218c216
-<             self.library_dirs.append(
-<                 os.path.join(_build_clib.build_clib, 'lib'),
-<             )
----
->             self.library_dirs.insert(0, os.path.join(_build_clib.build_clib, 'lib'))
-EOF
-    patch setup.py setup.py.patch && \
-    patch setup_support.py setup_support.py.patch
-}
-
-coincurve_build ()
-{
-    if ! coincurve_patch_ignore_sys_libsecp; then
-        return 1
-    fi
-    if [[ -d "${jm_deps}/secp256k1-${secp256k1_version}" ]]; then
-        unlink ./libsecp256k1
-        ln -sf "${jm_source}/deps/secp256k1-${secp256k1_version}" ./libsecp256k1
-    else
-        return 1
-    fi
-    COINCURVE_IGNORE_SYSTEM_LIB="1" python setup.py install
-    return "$?"
-}
-
-coincurve_install ()
-{
-    coincurve_version='9.0.0'
-    coincurve_lib_tar="${coincurve_version}.tar.gz"
-    coincurve_lib_sha='81561e954b4a978231e6611ae6153740bfbaebb214caff7a7b4e71fe9affbe09'
-    coincurve_url='https://github.com/ofek/coincurve/archive'
-
-    rm -rf "./coincurve-${coincurve_version}"
-    if ! dep_get "${coincurve_lib_tar}" "${coincurve_lib_sha}" "${coincurve_url}"; then
-        return 1
-    fi
-    pushd "coincurve-${coincurve_version}"
-    if ! coincurve_build; then
-        return 1
-    fi
-    popd
-}
-
-libsecp256k1_install ()
-{
-    secp256k1_version='1e6f1f5ad5e7f1e3ef79313ec02023902bf8175c'
-    secp256k1_lib_tar="${secp256k1_version}.tar.gz"
-    secp256k1_lib_sha='d4bc033398d4db43077ceb3aa50bb2f7700bdf3fc6eb95b8c799ff6f657a804a'
-    secp256k1_url='https://github.com/bitcoin-core/secp256k1/archive'
-
-    if check_skip_build "secp256k1-${secp256k1_version}"; then
-        return 0
-    fi
-    if ! dep_get "${secp256k1_lib_tar}" "${secp256k1_lib_sha}" "${secp256k1_url}"; then
-        return 1
-    fi
-    if ! coincurve_install; then
-        return 1
-    fi
-}
-
 libsodium_build ()
 {
     make uninstall
```

Comment:
or maybe do the `libsodium` install here?

Reply:
we install it after `libffi` on line `403` :)